community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v2_3/GeneratedQueryStructure.scala

```diff
@@ -714,6 +714,13 @@ private case class Method(fields: Fields, generator: CodeBlock, aux:AuxGenerator
     }
   }
 
+  override def indexUniqueSeek(nodeVar: String, descriptorVar: String, value: Expression) = {
```

Comment:
again plan is called `NodeIndexUniqueSeek` the only place where we use the other order is in the plandescription. I had to change to that above so that we conform to interpreted runtime.